manifests/gcp_marketplace/chart/kubeflow-pipelines/values.yaml

```diff
@@ -1,18 +1,18 @@
 images:
-  apiserver: gcr.io/ml-pipeline/api-server:0.1.31
-  argoexecutor: gcr.io/ml-pipeline/argoexec:v2.3.0-license-compliance
-  argoworkflowcontroller: gcr.io/ml-pipeline/workflow-controller:v2.3.0-license-compliance
-  cloudsqlproxy: gcr.io/cloudsql-docker/gce-proxy:1.14
-  frontend: gcr.io/ml-pipeline/frontend:0.1.31
-  metadataserver: gcr.io/tfx-oss-public/ml_metadata_store_server:0.14.0
-  minio: gcr.io/ml-pipeline/minio:RELEASE.2019-08-14T20-37-41Z-license-compliance
-  mysql: gcr.io/ml-pipeline/mysql:5.6
-  persistenceagent: gcr.io/ml-pipeline/persistenceagent:0.1.31
-  proxyagent: gcr.io/ml-pipeline/inverse-proxy-agent:0.1.31
-  scheduledworkflow: gcr.io/ml-pipeline/scheduledworkflow:0.1.31
-  viewercrd: gcr.io/ml-pipeline/viewer-crd-controller:0.1.31
-  visualizationserver: gcr.io/ml-pipeline/visualization-server:0.1.31
-  metadataenvoy: gcr.io/ml-pipeline/envoy:initial
+  apiserver: gcr.io/ml-pipeline/google/pipelines/apiserver:0.1.31
+  argoexecutor: gcr.io/ml-pipeline/google/pipelines/argoexecutor:0.1.31
```

Comment:
Reminder, 0.1.31 has not included licenses. We should first release 0.1.32 and then make this change.

Reply:
That might be too late for the alpha launch. Might be we can re-tagged the images so that the images included here contains proper licenses?
CC @rmgogogo

Reply:
Sure, SG since we have a time limit.

Reply:
I were told this file actually didn't take affects in MKP. Sorry for late reply. I should bind my @google.com with github